eng/tools/spec-gen-sdk-runner/src/types.ts

```diff
@@ -59,13 +59,43 @@ export type SdkName =
   | "azure-sdk-for-net"
   | "azure-sdk-for-python";
 
+/**
+ * Represents the plane types for SDK generation settings
+ */
+export interface PlaneTypeSettings {
```

Comment:
As discussed, I will enhance this type when need to support the separation of TypeSpec and OpenAPI specs.